src/Raven.Server/Documents/DocumentsStorage.cs

```diff
@@ -1767,8 +1767,7 @@ public static LazyStringValue TableValueToId(JsonOperationContext context, int i
         {
             // See format of the lazy string ID in the GetLowerIdSliceAndStorageKey method
```

Comment:
Comment needs to move to `GetLazyStringValue` method

Reply:
done